app/schemas/metric.py

```diff
@@ -1,21 +1,32 @@
-from typing import Optional
+from typing import Optional, List 
 
-from pydantic import BaseModel
+from pydantic import BaseModel, validator
 
 
 # Shared properties
 class MetricBase(BaseModel):
     name:  Optional[str] = None
     code:  Optional[str] = None
-    area:  Optional[str] = None
+    area:  Optional[List[str]] = None
     description: Optional[str] = None
     type: Optional[str] = None
 
+    
+
 # Properties to receive on metric creation
 class MetricCreate(MetricBase):
     name: str
     description: str
 
+    @validator('area', pre=True, always=True)
```

Comment:
praise: nice work for search for validators!
suggestion: treat empty list as if it's was passed as None inside the api route handler and don't throw a ValueError